tools/certs/Makefile

```diff
@@ -45,12 +44,18 @@ help: Makefile
 rawcluster := $(shell kubectl config current-context)
 cluster := $(subst /,-,$(rawcluster))
 pwd := $(shell pwd)
+export KUBECONFIG
+res := $(shell kubectl get secret istio-ca-secret  -n istio-system >/dev/null 2>&1;  echo $$?)
```

Comment:
this will run everytime, even when they don't call fetch-root-ca, which will mean only cluster admins can run these commands, even if its just to do offline things

Reply:
And getting the secret is a really, really bad thing to do - ok for a sample, but best for a sample of 'never do this'

Reply:
@costinm this is a request from you to add a function to fetch root ca from k8s  and sign certs using it :)